libkbfs/bare_root_metadata_v3.go

```diff
@@ -397,7 +395,18 @@ func (md *BareRootMetadataV3) IsWriter(
 		if err != nil {
 			return false, err
 		}
-		return wkb.IsWriter(user, deviceKey), nil
+		return wkb.IsWriter(user, cryptKey), nil
+	default:
+		panic(fmt.Sprintf("Unknown TLF type: %s", md.TlfID().Type()))
```

Comment:
I'm worried that a panic here will cause breakages if we introduce something other than `SingleTeam`, since this is called in a `default` case below. Wouldn't it be better to return an error?